src/customJs/tools/promo_code/index.ts

```diff
@@ -38,6 +38,21 @@ export const getPromoCodeToolDefinition: () =>
       default: {
         options: {
           alignment: alignmentProperty(),
+          backgroundColor: {
+            label: intl.formatMessage({ id: 'editor.background_color.label' }),
+            defaultValue: 'rgba(255,255,255, 0)',
+            widget: 'color_picker',
+          },
+          textColor: {
+            label: intl.formatMessage({ id: 'editor.font_color.label' }),
+            defaultValue: '#333333',
+            widget: 'color_picker',
+          },
+          fontSize: {
+            label: intl.formatMessage({ id: 'editor.font_size.label' }),
+            defaultValue: 36,
+            widget: 'counter',
+          },
```

Comment:
If I am not wrong, there is a `font_size` widget, did you try it? is it the same?

```suggestion
          fontSize: {
            label: intl.formatMessage({ id: 'editor.font_size.label' }),
            defaultValue: 36,
            widget: 'font_size',
          },
```

Reply:
I changed it, and I can't see the view change. Anyway, I leave as a font_size widget